reference/dic_tags.rst

```diff
@@ -1071,12 +1071,13 @@ the service is auto-registered and auto-tagged. But, you can also register it ma
         $container
             ->register(AppExtension::class)
             ->addTag('twig.extension')
-
+        ;
+        $container
```

Comment:
`addTag()` returns the `Definition` instance itself, but we need the `ContainerBuilder` to call `register()`